packages/block-library/src/navigation-submenu/index.php

```diff
@@ -199,9 +201,9 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 	// If Submenus open on hover, we render an anchor tag with attributes.
 	// If submenu icons are set to show, we also render a submenu button, so the submenu can be opened on click.
 	if ( ! $open_on_click ) {
-		$item_url = isset( $attributes['url'] ) ? esc_url( $attributes['url'] ) : '';
+		$item_url = isset( $attributes['url'] ) ? $attributes['url'] : '';
```

Comment:
This is incorrect because the escaping is done two lines below

Reply:
That's incorrect. This change is _removing_ the `esc_url` here and moving it to "two lines below" as per the aim of escaping late. That's correct.

Reply:
Argh my bad!

Reply:
No worries. Easily done!